python_src/tests/performance_manager/test_performance_manager.py

```diff
@@ -65,6 +65,7 @@
 from ..test_resources import springboard_dir, test_files_dir, csv_to_vp_parquet
 
 
+@pytest.mark.skip("utility function, not a test")
```

Comment:
`pytest` always threw a warning when running this function. this seems to clean that up.